Gemfile

```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem 'rspec-rails', '~> 3.5'
   gem 'simplecov'
   gem 'simplecov-rcov'
+  gem 'pry-rails'
```

Comment:
This should probably be its own PR. 

if this commit were on its own I could merge it. Then you could rebase all of your other branches on top of master to get pry in your test environment for all your branches